### PR TITLE
Endoc 165 cache

### PR DESCRIPTION
**Pull request by joewhite101**

### CODE REVIEW

**Comment by joewhite101** [comment]
The redis tutorial is not yet fully tested but the content can be reviewed generally.

File: vuepress/docs/next/docs/reference/caching-and-clustering.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+- In a read only implementation, or an implementation with infrequent writes to the cached objects listed above, the network latency between pods on different nodes will not be a major driver of runtime performance. Each pod will have a fully replicated copy of the cache
+- In write heavy implementations network latency between nodes can be a factor in performance
+  - The overall performance impact of network latency will vary depending on the implementation. The performance depends on the types of objects being written, the size of those objects, and whether the writes invalidate single objects or entire lists of objects in the cache.
+
+In general, it is recommended that performance testing on clustered instances matches the expected runtime traffic pattern of a live application. Every application will have a unique performance profile.
+
+### Cache Management
+
+When a new replica of an Entando App joins a cluster of applications the cache
```
Comment (nshaw) [inline vuepress/docs/next/docs/reference/caching-and-clustering.md:70]: This area looks incomplete or truncated?

**Review by nshaw — CHANGES_REQUESTED** [review]
I also sent a PR with a handful of minor wordsmithing changes.  I just reviewed the content, didn't try out the steps at this point.

File: vuepress/docs/next/docs/reference/caching-and-clustering.md (hunk 1 above)
Comment (joewhite101) [reply vuepress/docs/next/docs/reference/caching-and-clustering.md:70]: Done

Comment (nshaw) [reply vuepress/docs/next/docs/reference/caching-and-clustering.md:70]: @joewhite101 , did you push this change?

**Review by nshaw — APPROVED** [review]

**Pull request merged**